.circleci/config.yml

```diff
@@ -276,6 +276,20 @@ jobs:
           command: "cd tdrs-frontend; ./zap-scanner.sh"
       - store_artifacts:
           path: tdrs-frontend/reports/owasp_report.html
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/homepage.png
+
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/edit-profile.png
+
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/request.png
```

Comment:
@riatzukiza is there a meaningful distinction between this and the homepage?

Reply:
yes, this is the page the user is directed to after they have entered their name and stt in /edit-profile. This PR doesn't login to the app, so all of the endpoints that require authentication show the home page with a message telling the user to login. I made a new issue to set up pa11y to login to the app using a puppeteer browser instance.